### PR TITLE
Edit README.md on "bundle exec" for rails and add --binding option for rails server

### PR DESCRIPTION
**Pull request by tfaruq**
I edit README.md on "bundle exec" for rails (reference: http://stackoverflow.com/questions/23846493/what-is-the-difference-between-rails-s-and-bundle-exec-rails-s) and add --binding option when starting rails server with IP address to help newcomer in rails community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,13 @@ bundle exec rake db:migrate
 Finally start the rails server
 
 ```
-bundle exec rails s
+rails s
+````
+
+Or if you want to bind it with IP address (for example `--binding=127.0.0.1`)
```
Comment (BenMorganIO) [inline README.md]: Missing preposition. Probably "bind it with _an_ IP address".

**Comment by gmacdougall** [comment]
I don't think this is particularly necessary. We're assuming that people have a familiarity with rails and the options available there.

**Comment by Senjai** [comment]
I agree with gmac, we shouldn't be documenting rails in solidus.

**Comment by tvdeyen** [comment]
Yes, I also agree with @gmacdougall Especially if we start to document very specific stuff like binding IP addresses.

**Comment by cbrunsdon** [comment]
Thanks @tfaruq, but I'm going to close this out. I agree with the others that Rails experience is required and we're not looking to document rails' options in our readme.

Also, removing the "bundle exec" on the rails is less likely to work as not everyone installs rails globally, but everyone should have bundle globally.

**Comment by tvdeyen** [comment]
:+1: @cbrunsdon regarding `bundle exec`. Rails promotes `bin/rails`, since this handles binstubs and such. But this is for another PR

**Comment by tfaruq** [comment]
Thank you for the feedback!

**Pull request closed without merging**